test/e2e/tests/test_role.py

```diff
@@ -212,8 +212,40 @@ def test_crud(self, simple_role):
 
         cr = k8s.get_resource(ref)
         assert cr is not None
-        assert 'spec' in cr
-        assert 'inlinePolicies' not in cr['spec']
+        assert "spec" in cr
+        assert "inlinePolicies" not in cr["spec"]
 
         latest_inline_policies = role.get_inline_policies(role_name)
         assert len(latest_inline_policies) == 0
+
+        # AssumeRolePolicy testing
```

Comment:
Perhaps put in an assertion here that the assume role policy doc is of an expected value *before* you attempt to update it?

Reply:
Yeah, absolutely. This commit was mainly to push remotely as a WIP as I needed to change from PC to Mac 😄 

I'll hopefully be working on this tonight